.gen/pipeline/pipeline/model_secret_type_response_fields.go

```diff
@@ -16,5 +16,7 @@ type SecretTypeResponseFields struct {
 
 	Required bool `json:"required,omitempty"`
 
+	IsPassword *bool `json:"isPassword,omitempty"`
```

Comment:
**Question**: What are we using the nullability for?
If it's unset then it's true, if its set to false then false, if it's set to true then true?

In this case I would propose trying to reverse the field and using it without nullability.
I'm trying to find a non-negated name for the reverse field, but failed so far.
If we can't I'm fine with this, just an idea.

In general I support sensitive default for false value and explicit turn-off.

(If I assumed it wrongly then please clarify it for me and let's talk about it.)

(If we change it, the references need to be updated as well.)

Reply:
Checked the CLI change and if I understood correctly we are using nil value equals false in behavior so I'm confused why is it nullable.

Reply:
We discussed in private my original assumption was right, the CLI code was mistaken.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/pull/3202/commits/fb1bf55e3d9609c04f7fca8eccb6cd573cbbe7bd).